### DISCUSSION TITLE
Does this plugin have a limit for how much text can it read?

### DISCUSSION THREAD

**Discussion by Sheriif99** [dlutton/flutter_tts#390] [Q&A] [answered]
I've been using this plugin to read pdf books aloud by extracting the text from pdf using pdf_text plugin, however i found that large pdfs about 1mb big wont get read.

**Comment by dlutton** [comment, marked as answer]
@Sheriif99 I don't think iOS has a limit, however android has a 4,000 character limit through it's native google TTS engine.
